### ISSUE TITLE
Confused by the insight and pipeline

### ISSUE THREAD

**Issue by USTCWzy** [RCHI-Lab/BodyMAP#1]
Hello Abhishek! Congradulations for your new CVPR! I am very interested in BodyMAP but I am still confused by the pipeline of BodyMap.

In the paper, you mentioned the importance of getting the appiled 3D pressure map and proposed a network to predict 3d human mesh and 3D pressure map simultaneously. **However, as the 3D pressure map ground truth is generated (projected) from the 2D pressure map collected by a pressure sensing mat (or simulation) and the 2D pressure map is one of the network inputs, why do we need to predict the 3D pressure map instead of simply predicting the human mesh and then projecting the 2D pressure map input to the human mesh in the same manner to obtain the 3D ground truth (which is a similar pipeline as Bodies at Rest and Bodypressure but has both input modalities)**? If the pressure distribution of predicted 3D pressure map is not consistent with the 2D input ( which is prone to occur), which one should I believe in?

In short, given that the 3D pressure map ground truths are generated from 2D pressure map during model training, why don't we directly generate 3D pressure map outputs from 2D inputs instead of predicting them through the network?

Thanks very much for your reply!

**Comment by Tandon-A** [comment]
Hello @USTCWzy,

Apologies for the delay on my end. 

This is a really important question. Please find my comments below. 

1. Our work on this paper started by further analyzing BodyPressure. We noticed that BodyPressure 3D pressure map estimates formed by projecting the predicted 2D pressure image on top of the predicted mesh were better than the ones formed by projecting the ground truth 2D pressure image onto the predicted mesh. My reasoning for this is that the mesh estimates are many times not in conjunction with the ground truth 2D pressure image. We reasoned that learning the pressure map jointly with the predicted mesh would better tune the mesh and pressure estimates to be in conjunction with each other. Alas, I have found the same to be true for BodyMAP, i.e. the predicted 3D pressure map is better than the 3D pressure map formed by projecting 2D pressure image on top of the predicted mesh. This empirical evidence leads to more importance to the 3D pressure maps predicted by the model, and calls for further analysis and possibly training with larger datasets. 

2. Though we used multiple modalities for the main model, our approach achieves good performance even with just depth modality. As pressure sensors are quite expensive, the ability to predict 3D pressure maps jointly with the mesh with just depth modality is super beneficial for deployment. 
We are also now exploring possibilities of using other modalities than 2D pressure image as input, allowing for higher performance and removing the expensive 2D pressure image inputs.  

3. As illustrated in the appendix section, BodyPressure requires the iterative projection at inference time, making it very slow to run. BodyMAP instead predicts the 3D pressure map in a single forward pass, achieving real-time performance. A real-time visualization of pressure onto the body can be helpful in nursing homes and such facilities, and potentially enhance the pressure injury prevention process. 

Best, 
Abhishek

**Comment by USTCWzy** [comment]
Thanks very much for your reply!
